public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -571,19 +571,25 @@ function AutogeneratedRootIndicator() {
   return <strong> Auto-generated policies</strong>;
 }
 
-const InheritedProperties: FC<{ properties: InhertitableProperties }> = ({ properties }) => (
+const InheritedProperties: FC<{ properties: InheritableProperties }> = ({ properties }) => (
   <HoverCard
     arrow
     placement="top"
     content={
       <Stack direction="row" gap={0.5}>
-        {Object.entries(properties).map(([key, value]) => (
-          <Label
-            key={key}
-            label={routePropertyToLabel(key)}
-            value={<Strong>{routePropertyToValue(key, value)}</Strong>}
-          />
-        ))}
+        {Object.entries(properties).map(([key, value]) => {
+          if (!value) {
```

Comment:
Since `value` can now also be `null` we need to account for that – if the properties were `undefined` we wouldn't even be going over them with `Object.entries()`.